source/blog/2017-07-21-rpm-ostree-v2017.7-released.html.md

```diff
@@ -0,0 +1,315 @@
+---
+title: rpm-ostree v2017.7 released
+author: jlebon
+date: 2017-07-24 14:00:00 UTC
+tags: rpm-ostree, rpm, ostree, atomic, atomic host
+comments: true
+published: true
+---
+
+[rpm-ostree](https://rpm-ostree.readthedocs.org/) is the
+hybrid image/package system that provides transactional
+upgrades on Atomic Host. Here are some highlights from
+version
+[v2017.7](https://github.com/projectatomic/rpm-ostree/releases/tag/v2017.7),
+including PolicyKit and experimental overrides support.
+
+READMORE
+
+As usual, you can test out this release by
+[rebasing your Fedora Atomic Host](https://fedoraproject.org/wiki/QA:Updates_Testing#Using_it_with_Atomic_Host_.28Fedora_24_and_later.29)
+onto the testing branch. Feel free to also leave karma in
+the pending Bodhi updates:
+
+- [Fedora 25 update](https://bodhi.fedoraproject.org/updates/FEDORA-2017-45adcbb9a3)
+- [Fedora 26 update](https://bodhi.fedoraproject.org/updates/FEDORA-2017-483c7979ee)
+
+Note that this is the final release on Fedora 25.
```

Comment:
It's basically a consequence of FAHC switching to F26. Our CI no longer tests on F25 (see https://github.com/projectatomic/rpm-ostree/pull/871) and I think both Colin and I switched to developing on F26 to use FAHC RPMs. In all likelihood, I don't think we'll be cutting another release before F25 is done for good (I guess depending on how long we keep the `updates --> stable` thing going). If you feel strongly about it, you can open an issue and we can discuss further there, maybe?